libweb3jsonrpc/Test.cpp

```diff
@@ -33,6 +38,107 @@ using namespace jsonrpc;
 
 Test::Test(eth::Client& _eth): m_eth(_eth) {}
 
+string prepareVersionString()
```

Comment:
This should be moved to a generic place I'd assume. The client should also print this on start.

Reply:
Also if not moved elsewhere, should be placed into the anonymous namespace.

Reply:
this one copied from test namespace. however libwe3jsonrpc should not have deps on test namespace

Reply:
I think it should be a generic helper available allover cpp-eth, what do you think @chfast and @gumb0 ?